kubernetes/istio.yaml

```diff
@@ -153,3 +204,73 @@ spec:
           configMap:
             name: mixer-config
 ---
+################################
+# Istio ingress controller
+################################
+apiVersion: v1
+kind: Service
+metadata:
+  name: istio-ingress-controller
+  labels:
+    infra: istio-ingress-controller
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+#   nodePort: 32000
+    name: http
+  selector:
+    infra: istio-ingress-controller
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: istio-ingress-controller
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        alpha.istio.io/sidecar: ignore
+      labels:
+        infra: istio-ingress-controller
+    spec:
+      containers:
+      - name: istio-ingress-controller
+        image: docker.io/istio/proxy:2017-04-13-01.05.21
+        args: ["proxy", "ingress", "-v", "2"]
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 80
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+---
+################################
+# Istio CA
+################################
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: istio-ca
```

Comment:
Can we keep this as a separate file? Users will want flexibility whether deploying CA, and in which namespace to deploy CA.